azurerm/internal/services/desktopvirtualization/tests/virtual_desktop_host_pool_resource_test.go

```diff
@@ -0,0 +1,158 @@
+package tests
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/desktopvirtualization/parse"
+)
+
+func TestAccAzureRMVirtualDesktopHostPool_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_virtual_desktop_host_pool", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMDesktopVirtualizationHostPoolDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMVirtualDesktopHostPool_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDesktopVirtualizationHostPoolExists(data.ResourceName),
+					resource.TestCheckResourceAttr(data.ResourceName, "tags.%", "0"),
+				),
+			},
+		},
+	})
+}
+
```

Comment:
Could we add a complete test that sets all optional properties, and a update test which goes basic -> complete -> basic